sdk/ml/azure-ai-ml/tests/dsl/e2etests/test_dsl_pipeline_samples.py

```diff
@@ -363,7 +363,7 @@ def test_data_transfer_copy_2urifolder_job_in_pipeline(self, client: MLClient) -
         )
 
         pipeline = data_transfer_job_in_pipeline()
-        assert_job_cancel(pipeline, client)
```

Comment:
what about add a parameter like `skip_cancel=False` for `assert_job_cancel`? it will bring some convenience if we'd like update related logic in the future

Reply:
agree on this point

Reply:
updated